web/packages/demo/webpack.config.js

```diff
@@ -1,7 +1,7 @@
 /* eslint-env node */
 
 const path = require("path");
-const CopyWebpackPlugin = require("copy-webpack-plugin");
+const CopyPlugin = require("copy-webpack-plugin");
```

Comment:
Why?

Reply:
This is shorter and aligns with the official docs: https://webpack.js.org/plugins/copy-webpack-plugin/.
I didn't want to do this before because `CleanWebpackPlugin` couldn't be renamed and this would have been inconsistent, but since then you removed it in f6f722222125e14def6812780387da9e3c1cf78e.